### DISCUSSION TITLE
[Logout Timer] Countdown timer

### DISCUSSION THREAD

**Discussion by Digiridoo911** [runelite/runelite#15740] [Ideas]
I'd like to suggest a countdown timer that shows how much time you have left until you automatically logout.

If possible, it would be great if it could show in the window title as well:
https://puu.sh/JnuqV.png

**Comment by Bram91** [comment]
There is a hub plugin that does this:
https://runelite.net/plugin-hub/show/afk-countdown
Maybe this suits your needs?

**Reply by robrichardson13** [reply]
Not sure if that plug-in has been updated yet to account for the dynamic idle times?

**Reply by Digiridoo911** [reply]
Thanks, that it a very useful plugin 

It doesn't show the countdown time in the title bar yet, but I might suggest that there. Would be very useful for when Runelite is in the background
